src/Symfony/Component/Workflow/Definition.php

```diff
@@ -48,15 +48,15 @@ public function __construct(array $places, array $transitions, string|array $ini
     }
 
     /**
-     * @return string[]
+     * @return Place[]
```

Comment:
Switching from strings to your new class everywhere will break a lot of applications that currently use the component. You'll need to find another way.